src/setup.js

```diff
@@ -24,17 +24,25 @@ for (var i = 0, l = pixelFunc.queue.length; i < l; i++) {
   pixelFunc.process.apply(pixelFunc, pixelFunc.queue[i]);
 }
 
-window.addEventListener('unload', function() {
-  if (!Config.pageCloseOnce) {
+// https://github.com/GoogleChromeLabs/page-lifecycle/blob/master/src/Lifecycle.mjs
+// Safari does not reliably fire the `pagehide` or `visibilitychange`
+var isSafari = typeof safari === 'object' && safari.pushNotification;
+
+// IE9-10 do not support the pagehide event, so we fall back to unload
+// pagehide event is more reliable but less broad than unload event for mobile and modern browsers
+var pageCloseEvent = 'onpageshow' in self && !is_safari ? 'pagehide' : 'unload';
```

Comment:
Looks like a var name mismatch
```suggestion
var pageCloseEvent = 'onpageshow' in self && !isSafari ? 'pagehide' : 'unload';
```

Reply:
I would also love if we could pull out `'onpageshow' in self` as a variable similar to the isSafari one. For example something like this:
```jsx
var isNotIE = 'onpageshow' in self`
```

That way it will be more clear in the ternary what is happening:
```jsx
var pageCloseEvent = isNotIE && !isSafari ? 'pagehide' : 'unload';
```
Maybe even reverse `isSafari` to `isNotSafari` so it will be `isNotIE && isNotSafari` so it reads clear. Just matching  their naming would be great either positive or negative.

Reply:
You are right!
I did some improvements to better readability as you requested.